packages/config/src/utils.ts

```diff
@@ -0,0 +1,17 @@
+export const deepSearch = (
+  object: { [key: string]: any },
+  key: string,
+  predicate: (k: string, v: any) => boolean
+): any => {
```

Comment:
Can we get rid of _any_ here?

Reply:
It returns value and it can be anything.